index.html

```diff
@@ -51,8 +51,127 @@ <h1 class="intro-heading">Glad to see you!</h1>
+            </div>
+            <div class="textside">
+                <h2>Facebook 360</h2>
+                    <ul class="card-ul">
+                        <li>FACEBOOK</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>Experiment content creation feature that allows users to add an existing story over the course of a day without spamming their friends</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>Ruby on Rails</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
+            </div>  
+        </article>
+
+        <article class="project-articles">
+            <div class="textside">
+                <h2>Tonic</h2>
+                    <ul class="card-ul">
+                        <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>Experiment content creation feature that allows users to add an existing story over the course of a day without spamming their friends</p>
+                    <ul class="skills">
+                        <li>html</li>
+                        <li>Ruby on</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="see-project" href="#">See Project</a>
+ 
+            </div>
+            <div class="card">
+                <img src="/images/snapshot4.png">
+            </div>    
+        </article>
+
+    </section>
+
+    <section id="about">
+        <div>
+            <h2>About<br>Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <a href="#" class="social-para social-blue">LET'S CONNECT</a>                    
+                <ul>
+                    <li><a target="_blank" class="social-tag" href="https://www.github.com/joeveoper"><img src="/images/twitter.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.linkedin.com/in/yinka-olatunji-83aa36b5/"><img src="/images/linkedin.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.twitter.com/joeveoper"><img src="/images/medium.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.medium.com/"><img src="/images/github.png"></a></li>
+                    <li><a target="_blank" class="social-tag" href="https://www.angellist.com/"><img src="/images/angellist.png"></a></li>
```

Comment:
- [ ] Kindly add the correct color, font-weight, and font size of all the texts in your about section to match the figma design. Kindly look carefully to the CSS properties in the figma design 🥇

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219901407-b0163667-7ced-41e8-8987-fe847426e8a9.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/219901385-e0da8ccc-1335-4a71-bfd7-bd48a7001f63.png)